cmd/otelcontribcol/components.go

```diff
@@ -103,6 +104,7 @@ func components() (component.Factories, error) {
 		splunkhecreceiver.NewFactory(),
 		statsdreceiver.NewFactory(),
 		wavefrontreceiver.NewFactory(),
+		windowsperfcountersreceiver.NewFactory(),
```

Comment:
Per [guidelines](https://github.com/open-telemetry/opentelemetry-collector/blob/master/CONTRIBUTING.md#adding-new-components), should we enable the receiver in a subsequent PR?